xonsh/dirstack.py

```diff
@@ -5,28 +5,45 @@
 from glob import iglob
 from argparse import ArgumentParser
 
+from xonsh.tools import get_sep
+
 DIRSTACK = []
 """A list containing the currently remembered directories."""
 
+
 def _get_cwd():
     try:
         return os.getcwd()
     except (OSError, FileNotFoundError):
         return None
 
 
+def _splitpath(path, sofar=tuple()):
```

Comment:
better yet; i can remove this function.  i wrote it for a previous version, but it's not being used in this code.